types/apollo-upload-client/index.d.ts

```diff
@@ -1,22 +1,58 @@
-// Type definitions for apollo-upload-client 8.1
+// Type definitions for apollo-upload-client 14.1
 // Project: https://github.com/jaydenseric/apollo-upload-client#readme
-// Definitions by: Edward Sammut Alessi <https://github.com/Slessi>
+// Definitions by: Edward Sammut Alessi <https://github.com/Slessi>, tyankatsu <https://github.com/tyankatsu0105>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 3.1
+// TypeScript Version: 3.6
 
-import { ApolloLink } from "apollo-link";
-import { HttpOptions } from "apollo-link-http-common";
+// ==============================================================================
+// imports
+// ==============================================================================
 
-export { ReactNativeFile } from "extract-files";
+import { ApolloLink } from '@apollo/client/core';
+import { HttpOptions as OriginHttpOptions } from '@apollo/client/link/http';
+import { isExtractableFile as OriginIsExtractableFile } from 'extract-files';
```

Comment:
No need to rename this as you don't override it

Reply:
Wow. You're right.

Reply:
done